source/fundamentals/builders/sort.txt

```diff
@@ -29,38 +29,17 @@ Builders are classes provided by the MongoDB Java driver that help you construct
 :java-docs:`Bson <apidocs/bson/org/bson/conversions/Bson.html>` objects. 
 To learn more, see our :doc:`guide on builders </fundamentals/builders/>`.
 
-You should read this guide if you would like to:
-
-* Use builders to specify sort criteria for your queries.
-* Perform ascending sorts and descending sorts.
-* Combine sort criteria.
-* Sort on the text score of a
-  :manual:`text search </core/text-search-operators/>`.
+You should read this guide if you would like to use builders to specify sort
+criteria for your queries.
 
 If you want to learn the fundamentals of sorting in the MongoDB Java
 driver, consider reading our
 :doc:`guide on sorting </fundamentals/crud/read-operations/sort/>`.
 
-The Sorts Class
----------------
-
-The :java-docs:`Sorts </apidocs/mongodb-driver-core/com/mongodb/client/model/Sorts.html>`
-class is a builder that provides static factory methods for all sort criteria
-operators supported by MongoDB. These methods return a
-:java-docs:`Bson </apidocs/bson/org/bson/conversions/Bson.html>`
-object that you can pass to the 
-:java-docs:`sort() </apidocs/mongodb-driver-sync/com/mongodb/client/FindIterable.html#sort(org.bson.conversions.Bson)>`
-method of a ``FindIterable`` instance or to 
-:java-docs:`Aggregates.sort() </apidocs/mongodb-driver-core/com/mongodb/client/model/Aggregates.html#sort(org.bson.conversions.Bson)>`.
-If you want to learn more about ``Aggregates``, see our 
-:doc:`guide on the Aggregates builder </fundamentals/builders/aggregates>`.
-
-
 .. _sorts-builders-sort-example:
 
-The following examples show you how to use the methods
-provided by the ``Sorts`` class to sort your queries. The examples use a
-sample collection, ``sort_example``, that contains the following documents:
+The examples on this page use a sample collection, ``sort_example``, that
```

Comment:
Do we ever use the collection name "sort_example" in any of the snippets? I wasn't able to find any instances on this page, so maybe we can omit the name and just refer to it as the example collection?

Reply:
Thanks for the note. You are right, it is not referred to. Fix: https://github.com/mongodb/docs-java/pull/66/commits/3273cb0926c61881026ca6d759d4520857076100

Going forward, I will try to be more aware of what needs an explicit name, and try to be more aware of when names are helpful/unhelpful.